match/lib/match/runner.rb

```diff
@@ -71,6 +71,51 @@ def run(params)
       GitHelper.clear_changes
     end
 
+    def import_certificate(args, params)
+      UI.user_error!("It's not allowed to import your certificate because you enabled `readonly` option.") if params[:readonly]
+      UI.warning("Enterprise profiles are currently not officially supported in _match_, you might run into issues") if params[:type] == "enterprise"
+
+      params[:import_certificate] = args[0] if params[:import_certificate].nil?
+      UI.user_error!("Missing path to the certificate to import") if params[:import_certificate].nil?
+
+      params[:import_certificate_password] = args[1] if params[:import_certificate_password].nil?
```

Comment:
This currently breaks in the case where you specify `import_certficate` as an env var, but the password on the command line. Not sure how best to handle this though